src/Traits/ManagesItemsTrait.php

```diff
@@ -1,6 +1,9 @@
 <?php
 namespace Michaels\Manager\Traits;
 
+use Guzzle\Service\Exception\InconsistentClientTransferException;
```

Comment:
I don't see where this is used, but if it is, let's remove it. I'd rather not require any more dependencies.